src/arch/generic/pcstate.hh

```diff
@@ -126,6 +127,13 @@ class PCStateBase : public Serializable
         _upc = 0;
     }
 
+    virtual void
```

Comment:
Can you explain the need for this change? (I think I'm missing something)

Reply:
For the decoupled front-end I need a method to set the PC state to the start of a specific instruction address.
This should set the PC to the address passed in and reset the uop pc to 0.
Same is for `PCStateWithNext`. Here, I reset the next up pc to its initial value 1 (line 262)